psalm.xml

```diff
@@ -271,6 +271,11 @@
                 <referencedFunction name="Symfony\Component\PasswordHasher\PasswordHasherInterface::hash" />
             </errorLevel>
         </TooManyArguments>
+        <TooManyTemplateParams>
+            <errorLevel type="info">
+                <file name="src/Sylius/Bundle/UserBundle/Provider/UserProviderInterface.php" /> <!-- TODO: remove after dropping Symfony 5.4 -->
+            </errorLevel>
+        </TooManyTemplateParams>
```

Comment:
Here the same https://github.com/Sylius/Sylius/blob/1.13/psalm.xml#L190 :thinking: